src/game_battler.h

```diff
@@ -136,6 +136,15 @@ class Game_Battler {
 	 */
 	void ShiftAttributeRate(int attribute_id, int shift);
 
+	/**
+	 * Gets the current modifier (buff/debuff) to an attribute rate.
+	 * The shift is cleared after the battle ended.
+	 *
+	 * @param attribute_id Attribute modified
+	 * @return shift Shift applied.
+	 */
+	int GetShiftAttributeRate(int attribute_id);
```

Comment:
> 2.29: RM2000: Print "attribute resistance

Well the correct name would be "GetAttributeRateShift" because the original function is called "ShiftAttributeRate" because it shifts the attribute rate.